src/pages/Profile.js

```diff
@@ -1,87 +1,258 @@
 import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 import { ChartContainer } from '@codeforafrica/hurumap-ui';
+import gql from 'graphql-tag';
+import { useQuery, useApolloClient } from 'react-apollo-hooks';
+import { Grid } from '@material-ui/core';
 import { ProfilePageHeader } from '../components/Header';
 import ProfileTabs from '../components/ProfileTabs';
 import Page from '../components/Page';
 import CountryPartners from '../components/CountryPartners';
 import config from '../config';
-import { getProfile, getComparisonProfile } from '../lib/api';
 import ChartFactory from '../components/ChartFactory';
 import ChartsContainer from '../components/ChartsContainer';
 
+import sectionedCharts from '../data/charts.json';
+
 function Profile({
   match: {
-    params: { geoId, anotherGeoId }
+    params: { geoId, comparisonGeoId }
   }
 }) {
-  const head2head = Boolean(geoId && anotherGeoId);
-  const [activeTab, setActiveTab] = useState('');
-  const [selectedCountry, setSelectedCountry] = useState({});
-  const [profile, setProfile] = useState();
+  const head2head = Boolean(geoId && comparisonGeoId);
+  const [activeTab, setActiveTab] = useState('All');
+  const [profiles, setProfiles] = useState({
+    profile: {},
+    parentProfile: {},
+    comparisonProfile: {}
+  });
+  const client = useApolloClient();
+
+  // Provide the visuals with unique ids for fetching
+  // The unique ids will be used to set alias in graphql
+  let index = 0;
+  sectionedCharts.forEach(x =>
+    x.charts.forEach(y =>
+      y.visuals.forEach(z => {
+        // eslint-disable-next-line no-param-reassign
+        z.id = `viz${index}`;
+        index += 1;
+      })
```

Comment:
why don't we have unique ids defined in the json file?

Reply:
I decided to do this to reduce the amount of variables we set on the json file. I also think this way we can make the ids truly unique so we don't confuse when making the alias for the chart data. What are your thoughts @KhadijaMahanga ? You think we should go back to just specify them?

 cc @DavidLemayian